src/melee.cpp

```diff
@@ -1781,7 +1781,7 @@ std::string player::melee_special_effects( Creature &t, damage_instance &d, item
         sounds::sound( pos(), 16, sounds::sound_t::combat, "Crack!", true, "smash_success",
                        "smash_glass_contents" );
         // Dump its contents on the ground
-        weap.contents.spill_contents( pos() );
+        weap.spill_contents( pos() );
```

Comment:
```suggestion
        weap.contents.spill_contents( pos() );
```
this is actually functionally identical. i'll try to reduce code churn if i can here...